Engine/source/console/consoleObject.h

```diff
@@ -542,6 +545,19 @@ class AbstractClassRep : public ConsoleBaseType
 
 extern AbstractClassRep::FieldList sg_tempFieldList;
 
+inline AbstractClassRep* AbstractClassRep::getClassRepFromName(const char* className)
+{
+	AbstractClassRep* current = classLinkList;
+	while(current != NULL)
+	{
+		if(dStricmp(current->getClassName(),className) == 0)
```

Comment:
Oh wait, I should read more carefully. `dStricmp` is called inside a loop. AFAICT the string table only provides any benefit if you have a single `insert` and multiple comparisons. Otherwise the cost of `insert` will outweigh the cost of a comparison :P.